extensions/mssql/src/utils.ts

```diff
@@ -85,7 +85,7 @@ export function getCommonLaunchArgsAndCleanupOldLogFiles(prefix: string, executa
 	console.log(`This process (ui Extenstion Host) is pid: ${process.pid}`);
 	// Delete old log files
 	let deletedLogFiles = removeOldLogFiles(prefix);
-	console.log(`Old log files deletion report: ${JSON.stringify(deletedLogFiles)}`);
+	console.log(`Old log files deletion report: ${JSON.stringify(deletedLogFiles, undefined, '\t')}`);
 	launchArgs.push('--tracing-level');
```

Comment:
Why are you changing it? It seems unrelated to  your changes regardless

Reply:
This just pretty prints the deletedLogFiles information and makes it easier to read in the logs. Moving this fix out of this pull request nevertheless.

---
In reply to: [287893743](https://github.com/microsoft/azuredatastudio/pull/5499#discussion_r287893743) [](ancestors = 287893743)